src/main/java/org/kohsuke/github/OTPReceiverLambda.java

```diff
@@ -0,0 +1,7 @@
+package org.kohsuke.github;
+
+@FunctionalInterface
+public interface OTPReceiverLambda {
```

Comment:
Could this be a `[Supplier<String>](https://docs.oracle.com/javase/8/docs/api/java/util/function/Supplier.html)` instead? 
Is there a reason for it not to be?

Reply:
yup, i wasn't aware of the existing interface, changed.